src/views/portfolio/projects/ComponentVulnerabilities.vue

```diff
@@ -38,6 +38,24 @@
               return common.formatSourceLabel(row.source) + ` <a href="${url}">${xssFilters.inHTMLData(value)}</a>`;
             }
           },
+          {
+            title: this.$t('message.aliases'),
+            field: "aliases",
+            sortable: true,
```

Comment:
I don't think aliases are sortable, are they?

Reply:
Good point. They were marked as sortable in the existing alias column in ProjectFindings.vue which was copied over. But it doesn't work correctly, so i've disabled sorting in all 3 places now.